requirements.csv

```diff
@@ -24,3 +24,4 @@ anthonyharrison_not_in_db,lib4sbom
 anthonyharrison_not_in_db,lib4vex
 the_purl_authors_not_in_db,packageurl-python
 h2non,filetype
+python_packaging_authority_not_in_db,setuptools
```

Comment:
```suggestion
python,setuptools
```
There actually is a setuptools entry in nvd (I figured there had to be since snyk recommended pinning) so I looked it up for you.